edx_django_utils/monitoring/internal/backends.py

```diff
@@ -50,6 +50,13 @@ def record_exception(self):
         Record the exception that is currently being handled.
         """
 
+    @abstractmethod
+    def trace(self, name):
```

Comment:
Nitpicking on the naming... maybe something like this?

```suggestion
    def new_span(self, name):
```

I don't love this suggestion, but "trace" doesn't feel great either. I'd like to convey "we're opening/activating a span" but in particular I want to avoid any implication that we're creating a brand-new _trace_ (with a new root).

I never know what to name context managers anyhow. -.-